src/trace-mapping.ts

```diff
@@ -366,8 +416,8 @@ function OMapping(
   return { source, line, column, name } as any;
 }
 
-function GMapping(line: null, column: null): GeneratedMapping | InvalidGeneratedMapping;
-function GMapping(line: number, column: number): GeneratedMapping | InvalidGeneratedMapping;
+function GMapping(line: null, column: null): InvalidGeneratedMapping;
+function GMapping(line: number, column: number): GeneratedMapping;
```

Comment:
The lack of narrowed return type here seemed like it was probably unintentional?